src/reference_return_entity.md

```diff
@@ -165,4 +165,4 @@ optimization comes from the fact that the application developer can access this
 query parameter from the resource method, allowing them to conditionally avoid
 upstream service calls that would cause unnecessary slowdown.
 
-<a id="BATCH_PARTIAL_UPDATE"></a>
\ No newline at end of file
+<a id="BATCH_PARTIAL_UPDATE"></a>
```

Comment:
GH auto-adds a whitespace change when I'm editing directly on GH (tried it twice).
Verified from rich diff that there is no unintended change so this should be safe.